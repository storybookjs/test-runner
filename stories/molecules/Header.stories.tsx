import type { Meta, StoryObj } from '@storybook/react-vite';

import { Header } from './Header';

const meta = {
  component: Header,
  parameters: {
    layout: 'fullscreen',
  },
} satisfies Meta<typeof Header>;

export default meta;
type Story = StoryObj<typeof meta>;

export const LoggedIn: Story = {
  args: {
    user: {
      name: 'Jane Doe',
    },
  },
};

export const LoggedOut: Story = {};
