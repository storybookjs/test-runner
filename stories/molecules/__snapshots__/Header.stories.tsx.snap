// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Molecules/Header LoggedIn smoke-test 2`] = `
<header>
  <div class="storybook-header">
    <div>
      <svg width="32"
           height="32"
           viewbox="0 0 32 32"
           xmlns="http://www.w3.org/2000/svg"
      >
        <g fill="none"
           fill-rule="evenodd"
        >
          <path d="M10 0h12a10 10 0 0110 10v12a10 10 0 01-10 10H10A10 10 0 010 22V10A10 10 0 0110 0z"
                fill="#FFF"
          >
          </path>
          <path d="M5.3 10.6l10.4 6v11.1l-10.4-6v-11zm11.4-6.2l9.7 5.5-9.7 5.6V4.4z"
                fill="#555AB9"
          >
          </path>
          <path d="M27.2 10.6v11.2l-10.5 6V16.5l10.5-6zM15.7 4.4v11L6 10l9.7-5.5z"
                fill="#91BAF8"
          >
          </path>
        </g>
      </svg>
      <h1>
        Acme
      </h1>
    </div>
    <div>
      <span class="welcome">
        Welcome,
        <b>
          Jane Doe
        </b>
        !
      </span>
      <button type="button"
              class="storybook-button storybook-button--small storybook-button--secondary"
      >
        Log out
      </button>
    </div>
  </div>
</header>
`;

exports[`Molecules/Header LoggedOut smoke-test 2`] = `
<header>
  <div class="storybook-header">
    <div>
      <svg width="32"
           height="32"
           viewbox="0 0 32 32"
           xmlns="http://www.w3.org/2000/svg"
      >
        <g fill="none"
           fill-rule="evenodd"
        >
          <path d="M10 0h12a10 10 0 0110 10v12a10 10 0 01-10 10H10A10 10 0 010 22V10A10 10 0 0110 0z"
                fill="#FFF"
          >
          </path>
          <path d="M5.3 10.6l10.4 6v11.1l-10.4-6v-11zm11.4-6.2l9.7 5.5-9.7 5.6V4.4z"
                fill="#555AB9"
          >
          </path>
          <path d="M27.2 10.6v11.2l-10.5 6V16.5l10.5-6zM15.7 4.4v11L6 10l9.7-5.5z"
                fill="#91BAF8"
          >
          </path>
        </g>
      </svg>
      <h1>
        Acme
      </h1>
    </div>
    <div>
      <button type="button"
              class="storybook-button storybook-button--small storybook-button--secondary"
      >
        Log in
      </button>
      <button type="button"
              class="storybook-button storybook-button--small storybook-button--primary"
      >
        Sign up
      </button>
    </div>
  </div>
</header>
`;
